br/tests/config/tikv.toml

```diff
@@ -36,3 +36,6 @@ path = "/tmp/backup_restore_test/master-key-file"
 
 [log-backup]
 max-flush-interval = "50s"
+[gc]
```

Comment:
every time I run the test this is auto added to the toml file, let me know if I should remove it